### PR TITLE
make NativeMenuBar showing the Icons

### PR DESCRIPTION
**Pull request by Megapit**
## What does the pull request do?
make NativeMenuBar showing Icons under Windows

## What is the current behavior?
Under Windows 10 the NativeMenuBar is not showing the Icons. Using OSX the Icons are visible.
In the Control-Template the property "Icon" is missing. But NativeMenuItem.Icon is a IBitmap. MenuItem.Icon should be an Image
So i intruduced a converter from IBitmap to Image. Both Default and Fluent theme

## What is the updated/expected behavior with this PR?
Icons are shown under Windows

### CODE REVIEW

File: src/Avalonia.Themes.Fluent/Controls/NativeMenuBar.xaml [hunk 1]
```diff
@@ -4,12 +4,13 @@
        x:CompileBindings="True"
        Selector="NativeMenuBar">
   <Style.Resources>
-    <local:InverseBooleanValueConverter x:Key="AvaloniaThemesDefaultNativeMenuBarInverseBooleanValueConverter" Default="True"/>
+    <local:InverseBooleanValueConverter x:Key="AvaloniaThemesFluentNativeMenuBarInverseBooleanValueConverter" Default="True"/>
+    <local:IBitmapToImageConverter x:Key="AvaloniaThemesFluentNativeMenuBarIBitmapToImageConverterr"/>
```
Comment (maxkatz6) [inline src/Avalonia.Themes.Fluent/Controls/NativeMenuBar.xaml:8]:
```suggestion
    <local:IBitmapToImageConverter x:Key="AvaloniaThemesFluentNativeMenuBarIBitmapToImageConverter"/>
```
Typo. Also, in other places as well.

File: src/Avalonia.Themes.Fluent/Avalonia.Themes.Fluent.csproj [hunk 2]
```diff
@@ -10,6 +10,15 @@
...
   </ItemGroup>
+  <ItemGroup>
+    <None Remove="Controls\NativeMenuBar.xaml" />
+    <None Remove="NativeMenuBar.xaml" />
+  </ItemGroup>
+  <ItemGroup>
+    <AvaloniaResource Update="Controls\NativeMenuBar.xaml">
+      <Generator>MSBuild:Compile</Generator>
+    </AvaloniaResource>
+  </ItemGroup>
```
Comment (maxkatz6) [inline src/Avalonia.Themes.Fluent/Avalonia.Themes.Fluent.csproj:21]:
```suggestion
```

You don't need it. Just a garbage generated by Visual Studio.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0020160-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

File: src/Avalonia.Themes.Fluent/Controls/NativeMenuBar.xaml (hunk 1 above)
Comment (Megapit) [reply src/Avalonia.Themes.Fluent/Controls/NativeMenuBar.xaml:8]: @maxkatz6 
Upps... You are right :)

File: src/Avalonia.Themes.Fluent/Avalonia.Themes.Fluent.csproj (hunk 2 above)
Comment (Megapit) [reply src/Avalonia.Themes.Fluent/Avalonia.Themes.Fluent.csproj:21]: Sorry: My first ever Pull Request to an open-source projekt. Should have seen this.

**Comment by Megapit** [comment]
> You can test this PR using the following package version. `0.10.999-cibuild0020160-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

Perfect! Thanks for building. Now i can work on my litte Programm with Icons in the Menu 👍

**Comment by timunie** [comment]
> Perfect! Thanks for building. Now i can work on my litte Programm with Icons in the Menu 👍

this is just a bot showing you the link for testing. There is no little guy sitting and doing this for you 😉

Have a great weekend.

**Comment by Megapit** [comment]
> this is just a bot showing you the link for testing. There is no little guy sitting and doing this for you 😉

🙄 ahhh. ok
Like i said. I'm new to git and how it works in open source

File: src/Avalonia.Themes.Fluent/Controls/NativeMenuBar.xaml (hunk 1 above)
Comment (maxkatz6) [reply src/Avalonia.Themes.Fluent/Controls/NativeMenuBar.xaml:8]: @Megapit no problem. Can you commit fix of the typo?

Comment (Megapit) [reply src/Avalonia.Themes.Fluent/Controls/NativeMenuBar.xaml]: OK. Done
Did not know that i have to do this. I thought it is allready done by your suggestions.
Next time i know thw flow of making a pull request. I promise 😉

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `0.10.999-cibuild0020203-beta`. (feed url: https://nuget.avaloniaui.net/repository/avalonia-all/index.json) [PRBUILDID]

File: src/Avalonia.Themes.Fluent/Controls/NativeMenuBar.xaml (hunk 1 above)
Comment (maxkatz6) [reply src/Avalonia.Themes.Fluent/Controls/NativeMenuBar.xaml]: Thanks :)

**Review by maxkatz6 — APPROVED** [review]

File: src/Avalonia.Themes.Default/Controls/NativeMenuBar.xaml [hunk 3]
```diff
@@ -4,6 +4,7 @@
        Selector="NativeMenuBar">
   <Style.Resources>
     <local:InverseBooleanValueConverter x:Key="AvaloniaThemesDefaultNativeMenuBarInverseBooleanValueConverter" Default="True"/>
+    <local:IBitmapToImageConverter x:Key="AvaloniaThemesDefaultNativeMenuBarIBitmapToImageConverterr"/>
```
Comment (DyachenkoA) [inline src/Avalonia.Themes.Default/Controls/NativeMenuBar.xaml:7]: AvaloniaThemesDefaultNativeMenuBarIBitmapToImageConvert**er**

Comment (Megapit) [reply src/Avalonia.Themes.Default/Controls/NativeMenuBar.xaml:7]: Thanks. But already fixed in [https://github.com/AvaloniaUI/Avalonia/pull/8055/commits/6a8eb5a1cf551503c2090e3a0b8dfdc4eec8497c](url)

Comment (Megapit) [reply src/Avalonia.Themes.Default/Controls/NativeMenuBar.xaml:7]: > AvaloniaThemesDefaultNativeMenuBarIBitmapToImageConvert**er**

Sorry you are wright. Type still exists.
Will fix this with my next pull request for "isVisible" of NativeMenuItem

**Pull request merged**